zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -625,16 +625,28 @@ property, when available, is noted below.
     reaches a runtime generated random value in the \[snapCount/2+1, snapCount]
     range.The default snapCount is 100,000.
 
+* *snapSize* :
```

Comment:
I think name of this property should be consistent with `txnLogSizeLimitInKb`, because its purpose is similar.
What do you think about `snapSizeLimitInKb`?

Reply:
Sounds good to me, I don't think we need to operate on the granularity of individual bytes here save in testing.